tutorial/practicals/nemo/3_nemo_openmp/README.md

```diff
@@ -120,7 +103,19 @@ the number of MPI processes and resulting inter-process communication.)
    ```
    Since, by definition, a `CodeBlock` contains code that PSyclone does not
    understand, it will refuse to parallelise any region that contains
-   one.
+   one. If you perform a 'pass-through' with PSyclone (i.e. do not provide
+   an optimisation script) then you will see that the generated Fortran
+   will have comments added to it identifying any `CodeBlock` nodes and the
+   reason for them:
+
+   ```fortran
+   do jk = 1, jpk - 1, 1
+     do jj = 2, jpj - 1, 1
+       do ji = 2, jpi - 1, 1
+         ! PSyclone CodeBlock (unsupported code) reason:
+         !  - Unsupported statement: Write_Stmt
+         WRITE(4, *) mydomain(ji, jj, jk)
+   ```
```

Comment:
Great addition!!

Reply:
I can't take any credit for that - it's Sergi's :-)